packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -73,6 +73,8 @@ export function readExifData(file, buf) {
       new ExifImage({image: buf}, (error, exifData) => {
         if (!error && exifData) {
           file.image.orientation = exifData.image.Orientation;
+          file.image.width = exifData.exif.ExifImageWidth;
```

Comment:
Is this data populated by the server originally?

Reply:
it's populated by the clients originally. i meant when we get it back from the server